.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+src/main/java/java/resources/*
```

Comment:
In general, it is considered a good practice to include resources to gitignore file. But, in the task you have to create log4j2.xml/properties in this folder and we need to review it. Since you have included this file, I cannot do this.